elliottlib/errata.py

```diff
@@ -635,25 +635,21 @@ def add_jira_bugs_with_retry(advisory_id: int, bugids: List[str], noop: bool = F
         if noop:
             logger.info('Dry run: Would have attached bugs')
             continue
-        results = add_multi_jira_issues(advisory_id, chunk_of_bugs)
-        for i, result in enumerate(results):
-            if result.status_code != 201:
-                rt = add_jira_issue(advisory_id, chunk_of_bugs[i])
-                if rt.status_code != 201:
-                    errata_error = rt.json()
-                    if 'errors' not in errata_error or len(errata_error) != 1:
-                        raise exceptions.ElliottFatalError(f"attach jira bug {chunk_of_bugs[i]} failed with "
-                                                           f"status={rt.status_code} "
-                                                           f"errmsg={rt.json()}")
-                    if 'idsfixed' not in errata_error['errors'] or len(errata_error['errors']) != 1:
-                        raise exceptions.ElliottFatalError(f"attach jira bug {chunk_of_bugs[i]} failed with "
-                                                           f"status={rt.status_code} "
-                                                           f"errmsg={rt.json()}")
-                    for err in errata_error['errors']['idsfixed']:
-                        if 'The issue is filed already in' not in ' '.join(err):
-                            raise exceptions.ElliottFatalError(f"attach jira bug {chunk_of_bugs[i]} failed with "
-                                                               f"status={rt.status_code} "
-                                                               f"errmsg={rt.json()}")
+        try:
+            advisory.addJiraIssues(chunk_of_bugs)
+            advisory.commit()
+        except ErrataException as e:
+            logger.info(f"ErrataException Message: {e}\nRetrying...")
+            try:
```

Comment:
The unconditional retry logic here doesn't smell good. Would like to keep the original approach of handling exceptions or just let it crash if something wrong happens.